README.md

```diff
@@ -134,39 +132,29 @@ func main() {
 		Description: "Software Lib",
 	}
 
-	node1.Hashes = map[int32]string{
-		int32(sbom.HashAlgorithm_SHA1):   "f3ae11065cafc14e27a1410ae8be28e600bb8336",
-		int32(sbom.HashAlgorithm_SHA256): "4f232eeb99e1663d07f0af1af6ea262bf594934b694228e71fd8f159f9a19f32",
-		int32(sbom.HashAlgorithm_SHA512): "8044d0df34242699ad73bfe99b9ac3d6bbdaa4f8ebce1e23ee5c7f9fe59db8ad7b01fe94e886941793aee802008a35b05a30bc51426db796aa21e5e91b7ed9be",
-	}
+	node1.AddHash(sbom.HashAlgorithm_SHA1, "f3ae11065cafc14e27a1410ae8be28e600bb8336")
+	node1.AddHash(sbom.HashAlgorithm_SHA256, "4f232eeb99e1663d07f0af1af6ea262bf594934b694228e71fd8f159f9a19f32")
+	node1.AddHash(sbom.HashAlgorithm_SHA512, "8044d0df34242699ad73bfe99b9ac3d6bbdaa4f8ebce1e23ee5c7f9fe59db8ad7b01fe94e886941793aee802008a35b05a30bc51426db796aa21e5e91b7ed9be")
```

Comment:
At this moment, node1.AddHash causes panic because Hashes is not allocated in this example.  We should check for n.Hashes being nil in AddHash().  I'll suggest that below in node.go

Reply:
Or, the example could use NewNode().  But I still think node.go AddHash() should protect us against nil Hashes.

Reply:
Yeah this is why i missed it, I was testing only with nodes built with NewNode(). Thanks!